check.py

```diff
@@ -195,21 +195,14 @@ def fail_if_not_contained(actual, expected):
           raise Exception('wasm interpreter error: ' + err) # failed to pretty-print
         raise Exception('wasm interpreter error')
 
-print '\n[ checking binaryen-shell... ]\n'
```

Comment:
Ok, the reason this was tested is that the shell is the most user-facing tool here - I want stuff to break if we make that less useful for people :)